modules/common/istio/kiali-values.tpl

```diff
@@ -0,0 +1,9 @@
+kiali:
+  resources:
+    limits: 
+      cpu: 500m
+      memory: 128Mi
+    requests:
+      cpu: 30m
+      memory: 32Mi
+
```

Comment:
requests and limits for kiali are set here: https://github.com/liatrio/lead-terraform/blob/master/modules/common/istio/charts/kiali/templates/deployment.yaml#L42-L48